src/index.html

```diff
@@ -12,7 +12,72 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
+    <section class="stars stars--0">
```

Comment:
Add an empty line between sibling blocks of HTML. There should be an empty line before the <section> tag to separate it from the <body> tag.